features/mobile/not_safe_for_work.feature

```diff
@@ -0,0 +1,81 @@
+
+Scenario: Toggling nsfw state
+  #Nsfw users posts are marked nsfw
+  Given a nsfw user with email "tommy@pr0nking.com"
+  And a user with email "laura@officeworkers.com"
+  And a user with email "laura@officeworkers.com" is connected with "tommy@pr0nking.com"
+  And "tommy@pr0nking.com" has a public post with text "I love 0bj3ction4bl3 c0nt3nt!" and a poll
+  And "tommy@pr0nking.com" has a public post with text "I love 0bj3ction4bl3 c0nt3nt!" and a location
+  And "tommy@pr0nking.com" has a public post with text "I love 0bj3ction4bl3 c0nt3nt!" and a picture
+
+  #toggling nsfw state
+  When I sign in as "laura@officeworkers.com"
+  Then I should not see "I love 0bj3ction4bl3 c0nt3nt!"
+  And I should not see "What do you think about 1 ninjas?"
+  And I should not see "Posted from:"
+  And I should not see any picture in my stream
+  When I toggle all nsfw posts
+  Then I should see "I love 0bj3ction4bl3 c0nt3nt!"
+  And I should see "What do you think about 1 ninjas?"
+  And I should see "Posted from:"
+  And I should see 1 pictures in my stream
+
+Scenario: Resharing a nsfw post with poll
+  Given a nsfw user with email "tommy@pr0nking.com"
+  And a user with email "laura@officeworkers.com"
+  And a user with email "laura@officeworkers.com" is connected with "tommy@pr0nking.com"
+  And "tommy@pr0nking.com" has a public post with text "Sexy Senators Gone Wild!" and a poll
+  And I sign in as "laura@officeworkers.com"
+  And I toggle all nsfw posts
+  And I follow "Reshare"
+  And I confirm the alert
+  And I go to the home page
+  Then I should not see "Sexy Senators Gone Wild!"
+  And I should not see "What do you think about 1 ninjas?"
+  And I should have 2 nsfw posts
+
+Scenario: Resharing a nsfw post with location
+  Given a nsfw user with email "tommy@pr0nking.com"
+  And a user with email "laura@officeworkers.com"
+  And a user with email "laura@officeworkers.com" is connected with "tommy@pr0nking.com"
+  And "tommy@pr0nking.com" has a public post with text "Sexy Senators Gone Wild!" and a location
+  And I sign in as "laura@officeworkers.com"
+  And I toggle all nsfw posts
+  And I follow "Reshare"
+  And I confirm the alert
+  And I go to the home page
+  Then I should not see "Sexy Senators Gone Wild!"
+  And I should not see "Posted from:"
+  And I should have 2 nsfw posts
+
+Scenario: Resharing a nsfw post with picture
+  Given a nsfw user with email "tommy@pr0nking.com"
+  And a user with email "laura@officeworkers.com"
+  And a user with email "laura@officeworkers.com" is connected with "tommy@pr0nking.com"
+  And "tommy@pr0nking.com" has a public post with text "Sexy Senators Gone Wild!" and a picture
+  And I sign in as "laura@officeworkers.com"
+  And I toggle all nsfw posts
+  And I follow "Reshare"
+  And I confirm the alert
+  And I go to the home page
+  Then I should not see "Sexy Senators Gone Wild!"
+  And I should not see any picture in my stream
+  And I should have 2 nsfw posts
```

Comment:
`Scenario`s should be indented towards their `Feature`. Please extract the repetitive setup into a `Feature` level `Background`.